db/queries/assessment_records/queries.py

```diff
@@ -576,8 +576,8 @@ def get_assessment_records_score_data_by_round_id(round_id, selected_fields=None
     default_fields = [
```

Comment:
It looks like there's one fund that specifies its own `score_fields` which if I understand correctly will override these "default" fields and their order. As its only one it might be worth making sure the score order is consistent there too https://github.com/communitiesuk/funding-service-design-assessment-store/blob/511558a60544efcee9074611141bba423e391b08/config/mappings/assessment_mapping_fund_round.py#L261-L268